internal/configs/virtualserver_test.go

```diff
@@ -754,7 +754,11 @@ func TestGenerateUpstream(t *testing.T) {
 		LBMethod: "random",
 	}
 
-	result := generateUpstream(name, "", endpoints, isPlus, &cfgParams)
+	upstream := conf_v1alpha1.Upstream{
```

Comment:
the Name field is not used in the generateUpstream. I suggest not setting it all:
```
upstream := conf_v1alpha1.Upstream{}
```
Also, please place that line between lines 734 and 735, so that the input parameters are initialized in the order they are present in the generateUpstream signature. This is for consistency:
```
	name := "test-upstream"
	upstream := conf_v1alpha1.Upstream{}
	endpoints := []string{
		"192.168.10.10:8080",
	}
        . . .
```
This comment also applies for `TestGenerateUpstreamForZeroEndpoints` function.